dev/com.ibm.ws.jsp/src/com/ibm/wsspi/jsp/taglib/config/TldPathConfig.java

```diff
@@ -62,6 +61,7 @@ public String getUri() {
     /**
      * Sets the uri for the tld
      * param string String - the uri for the tld
+     * If the TLD has a URI attribute, then this URI is ignored unless forceCustomURI() or configureWithBothURIs() is called.
```

Comment:
Is this a behavior change?  This seems to imply if someone is currently calling setURI that it would no longer work and they now need to call additional methods in the SPI to get it to function again.